Makefile

```diff
@@ -125,3 +125,29 @@ down-pg_translit:
 clean: down-redis down-pg_translit down-pg
 	find . -name __pycache__ -exec rm -rf {} +
 	find . -name "*.pyc" -exec rm -rf {} +
+
+LOCAL_RUN_ONCE_SERVICES := osmboundaries_importer osm_importer
+LOCAL_DB_SERVICES := frontenddatabase mediatordatabase world-database osmboundaries-database
+LOCAL_APPLICATION_STACK := nginx frontend mediator worker worker-exclusive conversionserviceredis
+LOCAL_DEPLOY_VERSION := latest
```

Comment:
This should be `latest`, otherwise `build --pull` fails in some cases (compared to `git describe --dirty` as used in our aliases).